client/pages/home.jsx

```diff
@@ -50,30 +63,106 @@ function TopNav(props) {
   );
 }
 
-function HomeBody(props) {
+class HomeBody extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      tripEntries: this.props.tripEntries
+    };
+  }
```

Comment:
It's best to avoid storing props in state, you can just access `this.props` if you need the data. There are only a few reasons to store your props in state, but they are pretty special cases.

Reply:
Okay! Thanks Tim! And I looked at the code again and realized I didn't need that component to be a class component... so I changed it.